### ISSUE TITLE
The desert seems awful green

### ISSUE THREAD

**Issue by spirillen** [widelands/widelands#6179]
**Describe the bug**
This is a very old bug, that have last for month

When you chose desert in random game, you get the most beautiful green fields...

**To reproduce**
1. Choose random game
2. Choose desert

**Expected behavior**
A big desert to the gardening center...

**Screenshots**
![image](https://github.com/widelands/widelands/assets/44526987/a5479a0d-3e20-418a-988b-334ea04be372)

![image](https://github.com/widelands/widelands/assets/44526987/d93523bf-d40a-423c-9e57-34a892c9639a)

![image](https://github.com/widelands/widelands/assets/44526987/d975ef28-a623-4653-9e1c-48c253463f6d)

**Version:**
 - OS: Ubuntu 22.04
 - Widelands Version: A very old bug, not related to a specific release and I have no records for when I first spotted it, as I was thinking you have spotted this one for your self.

**Additional context**

**Comment by frankystone** [comment]
I don't think this is a bug. The setting "Desert" means only that terrains from the desert are used, which corresponds to the fourth tab in the editor place terrain tool:

![desert_terrains](https://github.com/widelands/widelands/assets/6730556/a9947554-d89c-4b09-af7e-ba45f1f60e2f)

As you can see there are some green terrains. Additionally: Trees will grow only on those greenish terrains. If the map is covered with only sandy terrains the resulting map is not playable.

**Comment by spirillen** [comment]
> If the map is covered with only sandy terrains the resulting map is not playable.

Agreed, but there are no sands, only a few pith here and there.

-----------------

Another note regarding the balance with quarts...  there are not enough, as finding stone mines are hard and they contain to few quarts, result, I was only able to build 3 towers and 1 fortress, even I owned more than 50% of the map.

**Comment by tothxa** [comment]
> Another note regarding the balance with quarts... there are not enough, as finding stone mines are hard and they contain to few quarts, result, I was only able to build 3 towers and 1 fortress, even I owned more than 50% of the map.

Empty stone mines should give quartz with 15% chance (1 in 6 to 7 tries on average), higher than empty iron mines of any other tribes. Have you built any, or did you only build them where quartz was found?

I see that you set resources to high. If that didn't give enough mineable stones, then I consider that a bug in the random generator, because that affects at least the Empire too (and to a much smaller degree, Atlanteans).

**Comment by spirillen** [comment]
> did you only build them where quartz was found?

I only found two stone mines at all... so yes I did build there. However, I didn't take the 15% into consideration, as the mines was empty...
